Source/ConversationV1/Models/MessageRequest.swift

```diff
@@ -17,111 +17,78 @@
 import Foundation
 
 /** A request formatted for the Conversation service. */
-public struct MessageRequest: JSONDecodable, JSONEncodable {
+public struct MessageRequest {
 
-    private let input: Input?
-    private let alternateIntents: Bool?
-    private let context: Context?
-    private let entities: [Entity]?
-    private let intents: [Intent]?
-    private let output: Output?
+    /// An input object that includes the input text.
+    public var input: InputData?
+
+    /// Whether to return more than one intent. Set to `true` to return all matching intents.
+    public var alternateIntents: Bool?
+
+    /// State information for the conversation. Continue a conversation by including the context object from the previous response.
+    public var context: Context?
+
+    /// Include the entities from the previous response when they do not need to change and to prevent Watson from trying to identify them.
+    public var entities: [RuntimeEntity]?
+
+    /// An array of name-confidence pairs for the user input. Include the intents from the previous response when they do not need to change and to prevent Watson from trying to identify them.
+    public var intents: [RuntimeIntent]?
+
+    /// System output. Include the output from the request when you have several requests within the same Dialog turn to pass back in the intermediate information.
+    public var output: OutputData?
 
     /**
-     Create a `MessageRequest`.
+     Initialize a `MessageRequest` with member variables.
 
      - parameter input: An input object that includes the input text.
-     - parameter alternateIntents: Whether to return more than one intent. Set to `true` to return
-        all matching intents. For example, return all intents when the confidence is not high
-        to allow users to choose their intent.
-     - parameter context: State information for the conversation. Include the context object from
-        the previous response when you send multiple requests for the same conversation.
-     - parameter entities: Include the entities from a previous response when they do not need to
-        change and to prevent Watson from trying to identify them.
-     - parameter intents: An array of name-confidence pairs for the user input. Include the intents
-        from the request when they do not need to change so that Watson does not try to identify
-        them.
-     - parameter output: System output. Include the output from the request when you have several
-        requests within the same Dialog turn to pass back in the intermediate information.
-     */
-    public init(
-        input: Input? = nil,
-        alternateIntents: Bool? = nil,
-        context: Context? = nil,
-        entities: [Entity]? = nil,
-        intents: [Intent]? = nil,
-        output: Output? = nil)
-    {
+     - parameter alternateIntents: Whether to return more than one intent. Set to `true` to return all matching intents.
```

Comment:
I guess the only change here is the formatting.  We need to look into reflowing comments (and maybe also code reformatting), but that's outside the scope of this PR.

Reply:
True. Opened an [issue](https://github.ibm.com/arf/planning-sdk-squad/issues/313).